### DISCUSSION TITLE
Detailed and Interactive Plot Visualization like WandB

### DISCUSSION THREAD

**Discussion by yusuke-intern** [langfuse/langfuse#2577] [Ideas]
### Describe the feature or potential improvement

Now I have to use WandB, to vizualize and analyze results on different models, prompts, in my experiment process.
Specifically, I want to see relationship between cost and accuracy, so I need to use WandB or other tools. so if I can do that on Langfuse, that would be very nice.

**Comment by marcklingen** [comment]
Can you share which kinds of views you find helpful in other tools or own workflows you built?

This might be a good first solution if you use Langfuse prompt management: https://langfuse.com/changelog/2024-05-16-prompts-version-insights

You can also use Langfuse datasets to benchmark different models and prompts: https://langfuse.com/docs/datasets/overview

**Comment by yusuke-intern** [comment]
Thank you so much for your support.
Unfortunately, I'm using LangChain.

**Comment by yusuke-intern** [comment]
<img width="1038" alt="image" src="https://github.com/langfuse/langfuse/assets/172477707/61b65680-d304-4146-8d93-3d691f649e2d">
I want a feature that allows me to easily change the horizontal and vertical axes under various conditions and group by model to display only the average values.
